source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -95,7 +95,7 @@ documentation:
 
 - `MongoCollection.watch() <{+api+}/apidocs/mongodb-driver-kotlin-coroutine/mongodb-driver-kotlin-coroutine/com.mongodb.kotlin.client.coroutine/-mongo-collection/watch.html>`__
 - `MongoDatabase.watch() <{+api+}/apidocs/mongodb-driver-kotlin-coroutine/mongodb-driver-kotlin-coroutine/com.mongodb.kotlin.client.coroutine/-mongo-database/watch.html>`__
-- `MongoClient.watch() <{+api+}/apidocs/mongodb-driver-kotlin-coroutine/mongodb-driver-kotlin-coroutine/com.mongodb.kotlin.client.coroutine/-mongo-client/watch.html>`__
+- `MongoClient.watch() <{+api+}/apidocs/mongodb-driver-kotlin-coroutine/mongodb-driver-kotlin-coroutine/com.mongodb.kotlin.client.coroutine/-mongo-client/index.html#1590990028%2FFunctions%2F-1409914714>`__
```

Comment:
I: I think the problem here is to fix the class name
```suggestion
- `MongoClient.watch() <{+api+}/apidocs/mongodb-driver-kotlin-coroutine/mongodb-driver-kotlin-coroutine/com.mongodb.kotlin.client.coroutine/-mongo-cluster/watch.html>`__
```

Reply:
@rustagir Think it's confusing to link to ``MongoCluster`` if the method call is for a ``MongoClient`` instance. What do you think?

Reply:
After discussion, will use the ``MongoCluster`` link for maintenance reasons